build-tools-internal/src/main/groovy/elasticsearch.formatting.gradle

```diff
@@ -196,7 +196,7 @@ def projectPathsToExclude = [
 
 subprojects {
   plugins.withType(ElasticsearchJavaPlugin).whenPluginAdded {
-    if (projectPathsToExclude.contains(project.path) == false) {
+    if (projectPathsToExclude.contains(project.path) == false || System.getProperty("es.format.everything") != null) {
```

Comment:
we should use `providers.systemProperty("es.format.everything").forUseAtConfigurationTime().isPresent()`
and not access system properties directly in gradle scripts